Framework/Geometry/test/ObjectTest.h

```diff
@@ -811,7 +811,7 @@ class ObjectTest : public CxxTest::TestSuite {
     // See
     // http://docs.mantidproject.org/nightly/concepts/HowToDefineGeometricShape.html#hexahedron
     Hexahedron hex;
-    hex.lbb = V3D(0, 0, 0);
+    hex.lbb = V3D(0, 0, -2);
```

Comment:
This is sensible, adjusting the limits to include negative portions of the z axis for checking the bounding box calculation. Good stuff.